credentials_manager/lib/credentials_manager/account_manager.rb

```diff
@@ -33,7 +33,8 @@ def user
     end
 
     def fetch_password_from_env
-      ENV["FASTLANE_PASSWORD"] || ENV["DELIVER_PASSWORD"]
+      password = ENV["FASTLANE_PASSWORD"] || ENV["DELIVER_PASSWORD"]
+      password unless password.to_s.length == 0
```

Comment:
Please add return statement here to be more explicit. Also it would be good to have an explicit `nil` return also, just to make sure it's clear for future developers

Reply:
Do you mean like this?
```
if password.to_s.length == 0
    return nil
else
    return password
end
```